test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -42,8 +42,13 @@ const (
 	cosOSImage        = "Container-Optimized OS from Google"
 	// Nvidia driver installation can take upwards of 5 minutes.
 	driverInstallTimeout = 10 * time.Minute
-	// Nvidia COS driver installer daemonset.
-	cosNvidiaDriverInstallerUrl = "https://raw.githubusercontent.com/ContainerEngine/accelerators/stable/cos-nvidia-gpu-installer/daemonset.yaml"
+)
+
+type podCreationFuncType func() *v1.Pod
+var (
+	gpuResourceName		v1.ResourceName
```

Comment:
I can't comment on line 63|68 but it should use `gpuResourceName` instead of `v1.ResourceNvidiaGPU`